### DISCUSSION TITLE
Galaxy Brain Badge

### DISCUSSION THREAD

**Discussion by ferin0079** [community/community#53893] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

## Galaxy Brain badge

Comment down below to get galaxy badge. First person to answer here gets the badge.

**Comment by Ferin79** [comment]
- Answer at least 2 discussions.
- Take note, you cannot mark your own answer as the answer to your own question.
- Answer questions asked by users in discussions and get marked.
- For example, if you mark this answer as "Mark as Answer", then I'll get a point for Galaxy Brain.
- Similarly, you will get a point when someone marks your answer as the answer in some discussion.

**Reply by queenofcorgis** [reply, marked as answer]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answering and locking the post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
